### PR TITLE
Removed global variabls from cpipe.c

### PR DESCRIPTION
**Pull request by kholis1**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [ ] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

Removed global variables from cpipe.c. Added these variables to cmd.c in every function that uses them to make the program thread safe.

...

**Test plan**

...

**Closing issues**

...

### CODE REVIEW

File: librz/cons/cpipe.c [hunk 1]
```diff
@@ -6,27 +6,26 @@
 
 // TODO: remove globals, and make this stackable
```
Comment (ret2libc) [inline librz/cons/cpipe.c:7]: remove comment

**Review by ret2libc — COMMENTED** [review]
Would it make sense to move those backup vars in RzCons or RzConsContext? RzCons is already one big global instance...

**Comment by kholis1** [comment]
I am a little bit confused. How should I proceed to eliminate those global variables?

**Comment by ret2libc** [comment]
> I am a little bit confused. How should I proceed to eliminate those global variables?

I think your approach was not wrong, it's just that maybe it is worth exploring other possibilities (like moving those two vars inside the RzConsInstance).

**Pull request closed without merging**